### PR TITLE
DESENG-72 Complete "files" section

### PR DESCRIPTION
**Pull request by Baelx**
[-] Wrote a test or tests for code you added.
[-] Linted the files you touched.
[x] Wrote a doc block for each method you added.
[-] Add documentation to the README

This is the final PR for this ticket. In it, I tried to make some improvements to code quality. While I still can't really write tests and I'm not sure if files are being linted properly, I at least tried to improve existing code adjacent to what I added.

- Add doc blocks for a whole controller.
- Remove console.logs
- Clean up redundant code

### CODE REVIEW

File: src/app/project/project-documents/project-document-table-rows/project-document-table-rows.component.ts [hunk 1]
```diff
@@ -18,11 +19,14 @@ export class DocumentTableRowsComponent implements OnInit, TableComponent {
   public paginationData: any;
 
   constructor(
+    public utils: Utils,
     private router: Router
   ) { }
 
   ngOnInit() {
     this.documents = this.data.data;
+    console.log('the data,', this.documents);
```
Comment (Baelx) [inline src/app/project/project-documents/project-document-table-rows/project-document-table-rows.component.ts]: I will get rid of this.

Comment (CR-gbc) [inline src/app/project/project-documents/project-document-table-rows/project-document-table-rows.component.ts]: this li'l logger 'sposta be here?

**Review by CR-gbc — APPROVED** [review]
might wanna wait for a Steve review, because my Angular is nowhere good as both of yours. but i didn't see much wrong with anything.

File: src/app/project/project-documents/detail/detail.component.html [hunk 2]
```diff
@@ -61,5 +61,10 @@ <h3 for="documentFileName">Filename: </h3>
       <h3 for="internalSize">Filesize: </h3>
       <p id="internalSize">{{ (document && document.internalSize) ? humanReadableSize : '0' }}</p>
     </div>
+    <div class="label-pair">
+      <h3 for="url">URL: </h3>
+      <a *ngIf="'Publish' !== publishText" id="url" target="blank" href="{{documentUrl || '-'}}">{{documentUrl || '-'}}</a>
+      <span *ngIf="'Publish' === publishText">-</span>
```
Comment (SteveMHoward) [inline src/app/project/project-documents/detail/detail.component.html]: Am I reading this right? If *not* published, show the file URL, and if published, show "-"?

Comment (Baelx) [reply src/app/project/project-documents/detail/detail.component.html]: So as with other places in the file, this checks for the state of the "publish" or "un-publish" button. The logic being that if you're currently able to publish, then the file must be un-published at that time. In retrospect it is confusing to read. I'll update it to use a more readable way to toggle the URL showing.

**Pull request merged**